core/opendaq/reader/src/multi_reader_impl.cpp

```diff
@@ -601,6 +615,9 @@ ErrCode MultiReaderImpl::skipSamples(SizeT* count, IMultiReaderStatus** status)
         return OPENDAQ_IGNORED;
     }
 
+    if (minReadCount > *count)
+        return makeErrorInfo(OPENDAQ_ERR_INVALIDPARAMETER, "Count parameter has to be larger than minReadCount.");
```

Comment:
Should we throw an exception in these cases, or simply return that we read no samples?